data-node/src/main/java/org/graylog/datanode/bootstrap/preflight/DataNodePreflightGeneratePeriodical.java

```diff
@@ -84,6 +84,7 @@ public DataNodePreflightGeneratePeriodical(final NodePreflightConfigService node
         this.keystoreStorage = keystoreStorage;
         // TODO: merge with real storage
         this.privateKeyEncryptedStorage = new PrivateKeyEncryptedFileStorage("privateKeyFilename.cert");
+        this.passwordSecret = passwordSecret;
```

Comment:
(Minor) I'd store` char[]` instead of `String`, to avoid conversion to `char[]` in every execution in `doRun()`.